src/metadataTextTracks.js

```diff
@@ -0,0 +1,123 @@
+import videojs from 'video.js';
```

Comment:
My goal for the suggestion is to let someone new to the project understand why this feature is useful. `cueTextTracks.js`is closer to that goal in my opinion, because "cues" are what the feature is intended to support.